src/c++/perf_analyzer/command_line_parser.cc

```diff
@@ -1438,18 +1438,20 @@ CLParser::VerifyOptions()
     params_->protocol = cb::ProtocolType::UNKNOWN;
   }
 
+  if (params_->should_collect_metrics && params_->enable_mpi) {
+    Usage(
+        "Cannot specify --collect-metrics when using the --enable-mpi "
+        "option. Server-size metric collection is only enabled when running "
```

Comment:
size->side

Also, for some reason I thought the warning we were going to add was related to multi-GPU, not multi-model. 

What happens for multi-model case? Are we not able to collect metrics? Is it possible to do so? Is there a story for it?  We are claiming multi-model support in MA, and we are now relying on PA for all GPU metrics.

Reply:
I think in the multi-model case there is duplicate work done by each instance of PA as each of them is gathering server side statistics which will be the same in the end. But I don't see why we can't enable GPU metrics for multi-model case.

Reply:
The Prometheus metrics support multi-GPU. It was the multi-model (i.e. PA MPI) case that we had concerns about functioning properly. But upon more consideration, I think yeah, it should be fine for each PA process to collect theoretically the same metrics and each output them however the user wants (e.g. stdout and/or csv). I've removed this particular check.

Reply:
I still see it? Did you push up your changes?